presentation/src/main/java/com/knocklock/presentation/lockscreen/mapper/NotificationMapper.kt

```diff
@@ -0,0 +1,157 @@
+package com.knocklock.presentation.lockscreen.mapper
+
+import android.app.PendingIntent
+import android.content.pm.ApplicationInfo
+import android.content.pm.PackageManager
+import android.graphics.drawable.Drawable
+import android.os.Build
+import android.service.notification.StatusBarNotification
+import com.knocklock.presentation.lockscreen.model.Notification
+import com.knocklock.presentation.util.TimeWithMeridiem
+import java.text.SimpleDateFormat
+import java.util.Date
+import java.util.Locale
+import com.knocklock.domain.model.Notification as NotificationModel
+
+/**
+ * @Created by 김현국 2023/03/07
+ */
+
+/**
+ * StatusBarNotification를 Array<[NotificationModel]>로 변환합니다.
+ *
+ * StatusBarNotification를 수정할 수 없기 때문에 List->Array로 변환합니다.
+ *
+ * @param statusBarNotifications Array<out [StatusBarNotification]> activeNotifications 입니다.
+ * @return Array<[NotificationModel]>
+ */
+fun toModel(statusBarNotifications: Array<out StatusBarNotification>, packageManager: PackageManager): Array<NotificationModel> {
+    return statusBarNotifications.asSequence().filter { it.toModel(packageManager) != null }.map { it.toModel(packageManager)!! }.toList().toTypedArray()
```

Comment:
toModel 연산이 2번 될 것 같아요! !!도 지울 수 있겠네요 :)
```suggestion
    return statusBarNotifications.asSequence().map{ it.toModel(packageManager) }.filter { it != null }.toList().toTypedArray()
```

Reply:
@jaeryo2357 [b4f68e1](https://github.com/Knock-Lock/Knock-Lock/pull/101/commits/b4f68e10bbc60762f74fede162e832aaa76f6b33)
Sequence라 mapping전 filter를 해주는게 좋아서, 기존 filter에서 model mapping되는 로직을 필터와 분리했습니다